ext/dropbear/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 3.14)
+project (dropbear C)
+
+if(DEFINED CMAKE_TOOLCHAIN_FILE)
+	set(CMAKE_C_FLAGS "-mfpu=neon -mfloat-abi=hard")
+endif()
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-function -Wno-unused-variable -ffunction-sections -fdata-sections -Wl,-gc-sections -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64")
+set(CMAKE_C_FLAGS_DEBUG "-g")
+set(CMAKE_C_FLAGS_RELEASE "-O3 -DNDEBUG")
+
+if(DEFINED CMAKE_TOOLCHAIN_FILE)
+	get_filename_component(COMPILER_NAME ${CMAKE_C_COMPILER} NAME)
+	get_filename_component(COMPILER_PATH ${CMAKE_C_COMPILER} PATH)
+	string(REGEX REPLACE "(.*)-gcc" "\\1" TARGET_PLATFORM "${COMPILER_NAME}")
+	set(TARGET_OPTIONS
+		"CFLAGS=${CMAKE_C_FLAGS}"
+		"CC=${COMPILER_PATH}/${TARGET_PLATFORM}-gcc"
+		"AR=${COMPILER_PATH}/${TARGET_PLATFORM}-ar"
+		"LD=${COMPILER_PATH}/${TARGET_PLATFORM}-ld"
+		"RANLIB=${COMPILER_PATH}/${TARGET_PLATFORM}-ranlib"
+		"NM=${COMPILER_PATH}/${TARGET_PLATFORM}-nm"
+		"AS=${COMPILER_PATH}/${TARGET_PLATFORM}-as"
+		"OBJDUMP=${COMPILER_PATH}/${TARGET_PLATFORM}-objdump"
+		"OBJCOPY=${COMPILER_PATH}/${TARGET_PLATFORM}-objcopy"
+		"STRIP=${COMPILER_PATH}/${TARGET_PLATFORM}-strip"
+		"STRINGS=${COMPILER_PATH}/${TARGET_PLATFORM}-strings"
+		"SIZE=${COMPILER_PATH}/${TARGET_PLATFORM}-size"
+	)
+	set(ENV ${TARGET_OPTIONS})
```

Comment:
do we know if this is scoped in any way? or is this manipulating the env for the lifespan of the cmake process?

Reply:
The later... performed the following tests:
Terminal 1: ran setup.sh and confirmed environment did not contain those environment variables.
Terminal 2: ran make -j from build directory and confirmed the environment variables were not present before or after.